docs/migration/2_0.md

```diff
@@ -198,12 +199,39 @@ The individual changes for 2.0 per class are:
 
 ### Pagination Component
 The reusable `PaginationComponent` has been completely refactored for 2.0. The pagination component had various flaws in version 1 and the implementation wasn't great either. The new version is fully configurable and easily extensible as the *build* logic is solely delegated to the new `PaginationBuilder`. 
-The default configuration is more conceise and shows maximum 3 pages with a start and end link. 
+The default configuration is more concise and shows maximum 3 pages with a start and end link. 
 
 The HTML and companied CSS is refactored as well. A clean DOM exists of only anchor links, nothing more. The availability and order of pagination links is driven by the configuration. The component is fully accessible and prepared for directionality as well. 
 
-Using anchor links is the prefered action for pagination links, but action links (usign `click` events) are still supported and used in various areas in Spartacus. The product listing page however is using anchor links.
+Using anchor links is the preferred action for pagination links, but action links (using `click` events) are still supported and used in various areas in Spartacus. The product listing page however is using anchor links.
 
 The component is fully [documented](https://github.com/SAP/cloud-commerce-spartacus-storefront-docs/blob/v2-develop/_pages/dev/components/shared-components/pagination.md).
 
 If you have used the pagination component directly, you should refactor the implementation, as the inputs have changed. 
+
+# Forms changes
```

Comment:
Would be good to also add here very simple example how to handle form validation now with `FormErrorsComponent`

Reply:
Done